src/pages/spawn/spawnHost/SpawnHostCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { Link } from "react-router-dom";
+import { SiderCard } from "components/styles";
+import { routes } from "constants/routes";
+import { Host } from "gql/generated/types";
+import { getDateCopy } from "utils/string";
+
+interface SpawnHostCardProps {
+  host: Host;
+}
+
+export const SpawnHostCard: React.FC<SpawnHostCardProps> = ({ host }) => (
+  <StyledSiderCard data-cy="spawn-host-card">
+    {Object.keys(spawnHostCardFieldMaps).map((key) => (
+      <SpawnHostEntry
+        key={`${key}_${spawnHostCardFieldMaps[key](host)}`}
+        field={key}
+        value={spawnHostCardFieldMaps[key](host)}
+      />
+    ))}
+  </StyledSiderCard>
+);
+
+interface SpawnHostEntryProps {
+  field: string;
+  value: string | { key: string; value: string }[];
+  type?: string;
+}
+const SpawnHostEntry: React.FC<SpawnHostEntryProps> = ({ field, value }) => (
+  <SpawnHostEntryWrapper>
+    <KeyWrapper>{field}</KeyWrapper>
+    <div>{value}</div>
+  </SpawnHostEntryWrapper>
+);
+
+const spawnHostCardFieldMaps = {
```

Comment:
can we change span tags to body tags except for the last key

Reply:
body tags?

Reply:
yup, there is `<Body /> ` tag in leafygreen that we use for text content
https://github.com/mongodb/leafygreen-ui/tree/master/packages/typography#body

Reply:
Thanks i confused it with the <body> tag